src/core/notifier/handler/notification/image_delete_handler.go

```diff
@@ -0,0 +1,24 @@
+package notification
+
+import (
+	"github.com/goharbor/harbor/src/pkg/notification/model"
+)
+
+// DeleteImagePreprocessHandler preprocess image delete event data
+type DeleteImagePreprocessHandler struct {
+}
+
+// Handle preprocess image delete event data and then publish hook event
+func (h *DeleteImagePreprocessHandler) Handle(value interface{}) error {
```

Comment:
merge handlers into one given eventType is in Value.

Reply:
> merge handlers into one given eventType is in Value.

fixed